src/app.js

```diff
@@ -1 +1,41 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { compareNumbers } = require('./compareNumber');
+const { generateNumber } = require('./generator');
+const { terminal } = require('./terminal');
+
+const generatedNumber = generateNumber();
+
+const checkUserNumber = (userNumber) => {
+  const isUserNumberValid = new Set(userNumber).size === 4
```

Comment:
When you use some static values like 4 here especially when you use it in two or more places, it's better to move it to some variable.